base/REPLCompletions.jl

```diff
@@ -167,8 +167,16 @@ function complete_path(path::AbstractString, pos; use_envpath=false)
             for file in filesinpath
                 # In a perfect world, we would filter on whether the file is executable
                 # here, or even on whether the current user can execute the file in question.
-                if startswith(file, prefix) && isfile(joinpath(pathdir, file))
-                    push!(matches, file)
+                if startswith(file, prefix)
+                    try
+                        # isfile can cause an error when trying to ask for stat on a
+                        # system file, it does not have permision to read.
+                        if isfile(joinpath(pathdir, file))
+                            push!(matches, file)
+                        end
+                    catch err
+                        isa(err, Base.UVError) || rethrow(err)
```

Comment:
I do not know what you mean by "can multiple happen". I can add the check for "EPERM", should I?